packages/core/src/App/Containers/RealityCheckModal/reality-check-modal.jsx

```diff
@@ -92,25 +156,33 @@ const RealityCheckModal = ({
                 validateForm={validateForm}
                 onSubmit={onSubmit}
                 logout={logoutClient}
-                reality_check_duration={reality_check_duration}
+                reality_check_duration={reality_check_duration || 0}
                 server_time={server_time}
-                IntervalField={IntervalField}
+                IntervalField={TradingViewIntervalField}
+            />
+        );
+    }
+
+    // Regards card https://redmine.deriv.cloud/issues/24171
```

Comment:
Because our code is open source, not all developers will be able to access the card on redmine. Maybe we can describe the issue here instead?